packages/ckeditor5-core/src/editor/editorui.js

```diff
@@ -53,6 +52,14 @@ export default class EditorUI {
 		 */
 		this.focusTracker = new FocusTracker();
 
+		/**
+		 * Stores viewport offsets from every direction.
```

Comment:
I'm afraid that this may be severely unclear :D What is a viewport offset? How it's used? Giving an example could be enough.